CHANGELOG.rst

```diff
@@ -4,6 +4,26 @@ Grafana Collection Release Notes
 
 .. contents:: Topics
 
+v2.1.0
+======
+
+Minor Changes
+-------------
+
+- Manage subfolders for `grafana_folder` and specify uid
+
+Deprecated Features
+-------------------
+
+- Deprecate `grafana_notification_channel` with removal in version 3.0.0
```

Comment:
I think that's just a nitpick, but we deprecate the module now and remove it in 3.0+0, right?

Reply:
yes this is right :)

Reply:
what would you change?

Reply:
It was late yesterday and I reread your text now. I somehow misread it...
what do you think about:
Deprecate `grafana_notification_channel`. It will be removed in version 3.0.0